example/widgetbook/pages/components/range_selector_widgetbook.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:zeta_flutter/zeta_flutter.dart';
+
+import '../../utils/scaffold.dart';
+import '../../utils/utils.dart';
+
+Widget rangeSelectorUseCase(BuildContext context) {
+  return WidgetbookScaffold(
+    builder: (context, _) => RangeSelectorExample(context),
+  );
+}
+
+class RangeSelectorExample extends StatelessWidget {
+  RangeSelectorExample(this.context);
+  final BuildContext context;
+
+  @override
+  Widget build(BuildContext _) {
+    return Padding(
+      padding: const EdgeInsets.all(16),
+      child: ZetaRangeSelector(
+        label: context.knobs.string(label: 'Label', initialValue: 'Range Selector'),
+        divisions: context.knobs.intOrNull.input(label: 'Divisions', initialValue: null),
+        showValues: context.knobs.boolean(label: 'Show Values', initialValue: true),
+        onChange: disabledKnob(context) ? null : (value) {},
+        largeSliderThumbs: context.knobs.boolean(label: 'Large Slider Thumbs', initialValue: false),
+        rounded: context.knobs.booleanOrNull(label: 'Rounded', initialValue: true),
```

Comment:
can you remove the rounded prop in widgetbook  - we already set rounded in the Addons panel and having this here overrides that